src/test/unit_tests/coretest.js

```diff
@@ -99,13 +99,12 @@ function setup(options = {}) {
     server_rpc.get_base_address = () => 'fcall://fcall';
 
     // register api servers and bg_worker servers locally too
+    // do not regsiter the object, func and common serivces as they are
```

Comment:
I don't understand how this works, isn't these services needed to be available on fcall://fcall for the coretest router?

Reply:
See my latest commit